lib/CXGN/Blast/Parse/Plugin/#Basic.pm#

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
should this buffer (#Basic.pm#) be added?

Reply:
no... that should be removed. Good catch. Will do.

Reply:
Done!